localstack/services/dynamodb/dynamodb_listener.py

```diff
@@ -430,23 +437,22 @@ def return_response(self, method, path, data, headers, response):
                     if existing_item:
                         record["dynamodb"]["OldImage"] = existing_item
 
-        elif action in ("GetItem", "Query"):
-            if response.status_code == 200:
-                content = json.loads(to_str(response.content))
-                # make sure we append 'ConsumedCapacity', which is properly
-                # returned by dynalite, but not by AWS's DynamoDBLocal
-                if "ConsumedCapacity" not in content and data.get("ReturnConsumedCapacity") in [
-                    "TOTAL",
-                    "INDEXES",
-                ]:
-                    content["ConsumedCapacity"] = {
-                        "TableName": table_name,
-                        "CapacityUnits": 5,  # TODO hardcoded
-                        "ReadCapacityUnits": 2,
-                        "WriteCapacityUnits": 3,
-                    }
-                    response._content = json.dumps(content)
-                    fix_headers_for_updated_response(response)
+        elif action in ("GetItem", "Query") and response.status_code == 200:
```

Comment:
Same as above - let's better leave the existing structure of the `if` blocks (to avoid unnecessary `elif` comparisons).